### ISSUE TITLE
[BUG] Missing fonts in docker container lead to some archived PDFs with garbled text

### ISSUE THREAD

**Issue by B5JohnSheridan** [paperless-ngx/paperless-ngx#6709]
### Description

I used so far Document Capture Pro from Epson to scan and OCR my documents. Those PDFs are imported by paperless (with OCR set to skip) with garbled text.

This is similar to #5839, but since already Ghostscript 10.02.1 is used, it is not like #5031.

At least in my case, it is realted to a missing package in the docker container, namely `fonts-droid-fallback`. As soon as it is installed, the PDFs are read correctly.

Once the font is installed, the ghostscript output changes from

```
Loading font OPExtFont0 (or substitute) from /usr/share/ghostscript/10.02.1/Resource/Font/NimbusSans-Regular
Loading font OPExtFont1 (or substitute) from /usr/share/ghostscript/10.02.1/Resource/Font/NimbusSans-Regular
```
to
```
Loading CIDFont TimesNewRomanPSMT substitute from /usr/share/ghostscript/10.02.1/Resource/CIDFSubst/DroidSansFallback.ttf
Loading CIDFont ArialMT substitute from /usr/share/ghostscript/10.02.1/Resource/CIDFSubst/DroidSansFallback.ttf
```

### Steps to reproduce

- Consume the attached PDF [test.pdf](https://github.com/paperless-ngx/paperless-ngx/files/15299912/test.pdf)
- Also possible to directly check with `gs -dBATCH -dNOPAUSE -sDEVICE=txtwrite -sFONTPATH=/usr/share/fonts/truetype/msttcorefonts -dFirstPage=1 -dLastPage=1 -o - test.pdf` inside webserver container (`docker exec -ti paperless-webserver-1 /bin/bash`)
- Install package `fonts-droid-fallback` in webserver container (`apt update; apt install fonts-droid-fallback`)
- Consume the attached PDF again or repeat test directly with ghostscript

### Webserver logs

```bash
No related log output, see above for ghostscript output when running manually.
```

### Paperless-ngx version

2.8.3

### Host OS

Ubuntu 24.04, current docker, ghcr.io docker images

### Installation method

Docker - official image

### Browser

Firefox current

### Please confirm the following

- [X] I believe this issue is a bug that affects all users of Paperless-ngx, not something specific to my installation.
- [X] I have already searched for relevant existing issues and discussions before opening this report.
- [X] I have updated the title field above with a concise description.

**Comment by stumpylog** [comment]
You're welcome to include the font yourself using [custom initialization](https://docs.paperless-ngx.com/advanced_usage/#custom-container-initialization), but we're never going to be able to include all the fonts a PDF may want to use.

**Comment by B5JohnSheridan** [comment]
Thanks for the quick reply!

I lack knowledge on the PDF internals, but since we are talking here about a text overlay from scanner OCR, that gets garbeled without the font, it is not so much about fonts needed by a PDF for nice looking. From what I was able to "google", it seems that this droid fallback font is indeed used quite often to resolve such issues, so it might still be worth considering if it could be added to the docker... 

Anyways, I already tried the custom initialization from a shorter explanation which lacked the rights stuff and did not work therefore. With your link, I got it working right away. So that solves this for me now, thanks a lot!
